### PR TITLE
Make the assignment names and icons standardized

### PR DESCRIPTION
**Pull request by tarebyte**
This also shows the name of the group of teams on the GroupAssignment Page.

Thought it might be useful.

/cc @johndbritton 

screen shot 2015-09-16 at 10 39 58

### CODE REVIEW

**Comment by johndbritton** [comment]
I don't see what changed with the icons, but LGTM :shipit:

> This also shows the name of the group of teams on the GroupAssignment Page.

Nice touch.

**Comment by johndbritton** [comment]
I like how the assignment listing page says "Individual assignment" and "Group assignment", I think each of the show pages should also have that written somewhere.

**Comment by tarebyte** [comment]
It does! 

Assignments show page: https://github.com/education/classroom/pull/220/files#diff-1124e712f8784939913e5eced551c957R15

The group assignments page is the screenshot

**Comment by tarebyte** [comment]
@johndbritton do you mean it should look like this?

```
Project Groups
Group Assignment
Project Groups
```

**Comment by johndbritton** [comment]
The screenshot above doesn't say "Group assignment" anywhere, the only indication is the icon.

I thought "Project Groups" was the name of the set of teams this assignment was distributed to.

**Comment by johndbritton** [comment]
You could do it as three lines in the header or move the grouping name down to just above the list of submissions

File: app/views/group_assignments/show.html.erb [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
-          <span class="assignment-icon assignment-icon-group">
-            <%= octicon('organization') %>
-          </span>
+        <span class="group-assignment-icon assignment-icon-group left">
+          <%= octicon('organization') %>
+        </span>
+        <h2 class="site-content-heading group-assignment-title">
           <%= @group_assignment.title %>
         </h2>
+        <p class="assignment-type text-muted"><%= @group_assignment.grouping.title %></p>
```
Comment (johndbritton) [inline app/views/group_assignments/show.html.erb]: How about updating to something like this:

> Group assignment for <strong><%= @group_assignment.grouping.title %></strong>

**Comment by tarebyte** [comment]
@johndbritton I forgot to style the pagination.

This is what it looks like now.

screen shot 2015-09-22 at 00 06 38

**Comment by tarebyte** [comment]
This is what the Group Assignment Page looks like

screen shot 2015-09-22 at 00 07 34

**Comment by tarebyte** [comment]
Assignment Listing

screen shot 2015-09-22 at 00 08 25

**Comment by johndbritton** [comment]
Perfect.

**Pull request merged**